pkg/frontend/frontend_test.go

```diff
@@ -282,14 +282,14 @@ func testFrontend(t *testing.T, config CombinedFrontendConfig, handler http.Hand
 	go httpServer.Serve(httpListen) //nolint:errcheck
 	go grpcServer.Serve(grpcListen) //nolint:errcheck
 
-	var worker services.Service
-	worker, err = frontend.NewWorker(workerConfig, querierConfig, httpgrpc_server.NewServer(handler), logger)
+	var wrkr services.Service
```

Comment:
If you name the import `querier_worker` like you did elsewhere, you can rollback the variable name to `worker`